src/app/core/components/manage-tags/manage-tags.component.spec.ts

```diff
@@ -117,7 +117,8 @@ describe('ManageTagsComponent #manage-tags', () => {
   });
 
   it('should not delete a tag if an error happens', async () => {
-    const { find, fixture, outputs } = await defaultRender();
+    const { find, fixture, instance, outputs } = await defaultRender();
+    instance.throwErrors = false;
```

Comment:
This wasn't the test that failed on #98; that was `can cancel out of renaming a tag`, below.

...which, now that I think about it, is kind of multiply concerning - isn't part of the point of that test that the delete endpoint does not get called?